lib/cast/cast_utils.js

```diff
@@ -339,6 +339,7 @@ shaka.cast.CastUtils.LargePlayerGetterMethods = {
   // NOTE: The 'getSharedConfiguration' property is not proxied as it would
   //       not be possible to share a reference.
   'getConfiguration': 4,
+  'getNonDefaultConfiguration': 4,
```

Comment:
Since this is only meant for issue reports, I think this would be better off not being in the cast messages at all, for the sake of efficiency.
Instead, I would just have it issue an error and use the local version, in `shaka.cast.CastProxy.playerProxyGet_`, like how we handle local-only methods like `getAdManager`.

Reply:
Done!